pkg/resourcemanager/predicate/has_operation_annotation.go

```diff
@@ -16,16 +16,17 @@ package predicate
 
 import (
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	predicateutils "github.com/gardener/gardener/pkg/controllerutils/predicate"
 
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 )
 
 // HasOperationAnnotation is a predicate for the operation annotation.
 func HasOperationAnnotation() predicate.Predicate {
-	return FromMapper(MapperFunc(func(e event.GenericEvent) bool {
+	return predicateutils.FromMapper(predicateutils.MapperFunc(func(e event.GenericEvent) bool {
```

Comment:
Nit: Do you want to eliminate the `FromMapper` usage here while you are on it?

Reply:
I would rather not do this as part of this PR and keep it in favor of #4487 given that there are many other usages of `FromMapper`.